### PR TITLE
Modify root.store file

### LINKED ISSUE #105: Implement check on production / development (for redux-devtools)

### PR DESCRIPTION
**Pull request by ivan-bonk**
fix problem with deployment on heroku
close #105

### CODE REVIEW

File: src/pages/comparison/components/remove-pet-button/remove-pet-button.component.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { useDispatch } from 'react-redux';
 import { Link } from 'react-router-dom';
 import { Profile } from '../profile/profile.component';
 import { RemoveButtonProps } from './remove-pet-button.interface';
-import { removeComparisonPet } from '../../../../store/comparison/actions/comparison-handler.action'
+import { removeComparisonPet } from '../../../../store/comparison/actions/comparison-handler.action';
```
Comment (okhabr) [inline src/pages/comparison/components/remove-pet-button/remove-pet-button.component.tsx]: FYI: you may import it like
'shared/store/comparison/actions/comparison-handler.action'

**Review by okhabr — APPROVED** [review]
👍

File: src/pages/comparison/components/remove-pet-button/remove-pet-button.component.tsx (hunk 1 above)
Comment (ivan-bonk) [reply src/pages/comparison/components/remove-pet-button/remove-pet-button.component.tsx]: right
forgot about it

**Review by Pashkaua — APPROVED** [review]

**Review by Danik1702 — APPROVED** [review]
Fine!)

**Pull request merged**